_posts/2016-07-20-Automation-testing-with-postman.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+authors: [greg_rinaldi]
+title: 'API Testing with Postman and Newman'
+image: /img/postman.png
+tags: [Postman,Integration Testing,Newman,Automation,Testing]
+category: Testing
+comments: true
+---
+### Prerequisites
+For the purpose of this tutorial it is mandatory to have [Postman](https://www.getpostman.com/){:target="_blank"} installed which is available as a Chrome extension, as well as a Mac application. And it is also mandatory to create an account at [Algorithmia](https://www.algorithmia.com/){:target="_blank"}.
+
+### Creating and selecting an environment
+Postman's environment functionality makes it very easy to switch between different environments. A set of variables can be configured per environment and when switching from one environment to the other these will be replaced accordingly. For example let’s create an environment called "production".  Click the "No environment" dropdown in the header and select "Manage environments".
```

Comment:
set of variables can be configured per environment and when switching from one environment to another one,